### ISSUE TITLE
Distortion type resets to Grill Distortion when UI is opened after project/session load

### ISSUE THREAD

**Issue by crshrprt** [Davit-G/Hamburger#8]
When a saved project/session is loaded Hamburger keeps the saved distortion type setting until UI is opened. After UI is opened it resets to Grill Distortion.

OS: Linux, Debian Sid
Tested on:
Ardour 8.10.0
Carla 2.5.9
BespokeSynth nightly

**Comment by Davit-G** [comment]
I wasn't able to replicate this unfortunately, so I'm not sure why this is happening. Will get back to this

**Comment by Davit-G** [comment]
Coming back to this, I was able to replicate this issue on Ubuntu 23.10 running Bespoke Synth 1.2.1.

Turns out every single dropdown / category element which was assigned to a plugin parameter had the same behaviour. I added some manual code to make sure that the distortion type (and other dropdown choices) are set correctly before linking the UI to the VST parameters themselves. I guess on some systems this doesn't happen automatically.

The latest commit (https://github.com/Davit-G/Hamburger/commit/6d7ecc097e8d00fa19124bea55edac97baaa1891) should have this issue fixed. Try recompiling and let me know if this fixes the issue.

Thanks!
